README.md

```diff
@@ -15,6 +24,42 @@ The steps below describe what needs to be done to use Platoon for
 data-parallelism. The LSTM example in the folder 'example' was implemented
 following these steps and should be referred to for guidance.
 
+Timing of the LSTM example on 2 k80
+-----------------------------------
+
+The timing is about efficiency of computation, not efficiency of
+training.  So the parameter alpha is constant. The number of minibatch
+is fixed as the hyper-parameter. The sync is also fixed to be after 10
+mini-batch of computation.
+
+With 1 worker, it won't train well. This isn't recommended. This is
```

Comment:
I updated it following the new review.

On Fri, Jan 8, 2016 at 11:45 AM carriepl <notifications@github.com> wrote:

> In README.md
> <https://github.com/mila-udem/platoon/pull/14#discussion_r49208995>:
>
> > @@ -15,6 +24,42 @@ The steps below describe what needs to be done to use Platoon for
> >  data-parallelism. The LSTM example in the folder 'example' was implemented
> >  following these steps and should be referred to for guidance.
> >
> > +Timing of the LSTM example on 2 k80
> > +-----------------------------
>
> ------
> > +
> > +The timing is about efficiency of computation, not efficiency of
> > +training.  So the parameter alpha is constant. The number of minibatch
> > +is fixed as the hyper-parameter. The sync is also fixed to be after 10
> > +mini-batch of computation.
> > +
> > +With 1 worker, it won't train well. This isn't recommended. This is
>
> "It won't train well" => Needs a bit more detail. Do you mean "The model
> performance is crappy" or "it will be slower than not using Platoon at all"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mila-udem/platoon/pull/14/files#r49208995>.
>